README.md

```diff
@@ -268,6 +268,57 @@ editors:
   - css_select: h1 > a
 ```
 
+#### `css_sort`
+
+Similar to [`css_select`](#css_select) but sorts the matched HTML elements before returning them.
+
+Takes both a [CSS Selector](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Selectors) and editors to sort by key.
+
+Examples:
+
+```yaml
+editors:
+  # Get all articles and sort them
+  - css_sort:
+      selector: article
+```
+
+The above example sorts by the whole element, so it includes stuff like HTML element attributes.
+In order to sort by a given key, editors can be used.
+
+```yaml
+editors:
+  # Get all articles and sort them by their heading
+  - css_sort:
+      selector: article
+      sort_by: # here you can use every editor again which is applied to every selected html element
+        - css_select: h2
+```
+
+This can still result in surprising results as the attributes are still included.
```

Comment:
What exactly is surprising?

Reply:
Also: `results in results` ;)

Reply:
312abfb94bf11cc015d7db5cca2b5362333623a3

Reply:
Oh that's so good! On point!